File: pset6/mario.py
from cs50 import get_int

while True:
    height = get_int("whats the height of the pyramid\n")
    if height < 1 or height > 8:
        print("height must be between 1 and 8, inclusive")
    else:
        break

for i in range(height):
    for m in range(height - i - 1):
        print(" ", end='')
    for j in range(i + 1):
        print("#", end='')
    print("  ", end='')
    for k in range(i + 1):
        print("#", end='')
    print("")

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int input = get_int("enter the height of the pyramid:");
    while (input < 1 || input > 8)
    {
        input = get_int("enter the height of the pyramid:");
    }
    int ip = input;
    for (int i = 0; i < input; i++)
    {
        for (int m = 0; m < ip - 1 ; m++)
        {
            printf(" ");
        }
        -- ip;
        for (int j = 0; j <= i ; j++)
        {
            printf("#");
        }
        printf("  ") ;       
        for (int n = 0; n <= i ; n++)
        {
            printf("#");
        }
        printf("\n");        
    }
}
